staging/src/k8s.io/apiserver/pkg/cel/openapi/maplist.go

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
same question about how we'll ensure staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/maplist.go and this file don't drift

Reply:
ditto